### PR TITLE
fix(k8s-website-setup-HA): typo

### PR DESCRIPTION
**Pull request by opera443399**
replace the `endpoint-reconciler-type=lease` extra argument with `endpoint-reconciler-type: lease`

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7137)

### CODE REVIEW

**Comment by opera443399** [comment]
> Some checks haven’t completed yet

*should I do something for it?*

I've no idea how to continue for this PR

File: docs/setup/independent/high-availability.md [hunk 1]
```diff
@@ -197,9 +196,9 @@ In order to copy certs between machines, you must enable SSH access for `scp`.
 
     ```shell
     cfssl print-defaults csr > config.json
-    sed -i '0,/CN/{s/example\.net/'"$PEER_NAME"'/}' config.json
-    sed -i 's/www\.example\.net/'"$PRIVATE_IP"'/' config.json
-    sed -i 's/example\.net/'"$PUBLIC_IP"'/' config.json
+    sed -i -e '0,/CN/{s/example\.net/'"$PEER_NAME"'/}' \
```
Comment (heckj) [inline docs/setup/independent/high-availability.md]: `config.json` was removed, which is rather important in this example... or it hasn’t been fulled updated to flow from the `cfssl print-defaults csr` inline. One or the other...

**Comment by heckj** [comment]
@opera443399 nothing further you need to do to deal with the checks, that’s simply the merge automation reporting that this PR hasn’t yet been reviewed and approved. Please do review the changes you made for correctness though - I don’t think merging that example into a single flow makes it more understandable, and doesn’t fit with the “fixing a typo” description you put in the PR message

File: docs/setup/independent/high-availability.md (hunk 1 above)
Comment (opera443399) [reply docs/setup/independent/high-availability.md]: @heckj 
agree with you

**Comment by opera443399** [comment]
@heckj thank you for your guide on this PR, as it's my first one

File: docs/setup/independent/high-availability.md [hunk 2]
```diff
@@ -443,7 +442,7 @@ Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**).
     - `<etcd0-ip>`, `<etcd1-ip>` and `<etcd2-ip>` with the IP addresses of your three etcd nodes
     - `<podCIDR>` with your Pod CIDR. Please read the [CNI network section](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network) of the docs for more information. Some CNI providers do not require a value to be set.
 
-    **Note:** If you are using Kubernetes 1.9+, you can replace the `apiserver-count: 3` extra argument with `endpoint-reconciler-type=lease`. For more information, see [the documentation](https://kubernetes.io/docs/admin/high-availability/#endpoint-reconciler).
+    **Note:** If you are using Kubernetes 1.9+, you can replace the `apiserver-count: 3` extra argument with `endpoint-reconciler-type: lease`. For more information, see [the documentation](https://kubernetes.io/docs/admin/high-availability/#endpoint-reconciler).
```
Comment (heckj) [inline docs/setup/independent/high-availability.md]: Was the switch from `endpoint-reconciler-type=lease` to `endpoint-reconciler-type: lease` intentional? It doesn’t match the documentation you reference at https://kubernetes.io/docs/admin/high-availability/#endpoint-reconciler, which uses the `=` format, and I suspect is correct.

**Comment by heckj** [comment]
@opera443399 we very much appreciate it! Thank you for taking the time to make the PR. I did find another potential issue - I apologize, I should have asked about it earlier in the first review pass. Would you take a look and verify the correctness of that other change?

File: docs/setup/independent/high-availability.md (hunk 2 above)
Comment (opera443399) [reply docs/setup/independent/high-availability.md]: @heckj I did not visit the reference, it's the original link.
as in my experiment with v1.9.0
`endpoint-reconciler-type: lease` is the right one.

Comment (opera443399) [reply docs/setup/independent/high-availability.md]: @heckj 
I guess:

> You can switch to the new reconciler by adding the flag --endpoint-reconciler-type=lease

which means the form:
kubeadm --param-1 --param-2

Comment (opera443399) [reply docs/setup/independent/high-availability.md]: please take a look at my [exp](https://github.com/opera443399/ops/blob/master/doc/CLOUD/k8s/k8s%E9%9B%86%E7%BE%A4%E9%83%A8%E7%BD%B2%E6%89%8B%E5%86%8C/%E5%86%8D%E6%8E%A2%E4%BD%BF%E7%94%A8kubeadm%E9%83%A8%E7%BD%B2%E9%AB%98%E5%8F%AF%E7%94%A8%E7%9A%84k8s%E9%9B%86%E7%BE%A4-01%E5%BC%95%E8%A8%80.md#%E9%85%8D%E7%BD%AE-k8s-master)

**Comment by heckj** [comment]
@opera443399 thanks for the explanation

**Comment by heckj** [comment]
@jbeda would you be willing to give this a review for technical accuracy please? If it looks good, please `lgtm` it in a comment and remove the hold and it'll get merged in.

/approve

File: docs/setup/independent/high-availability.md (hunk 2 above)
Comment (jbeda) [reply docs/setup/independent/high-availability.md]: This should stay `endpoint-reconciler-type: lease`.  The `--foo=bar` syntax is when the flag is passed on the command line.  In a YAML file it is `foo: bar`.  This section is talking about the kubeadm config yaml file and as such it should be `foo: bar`.

**Comment by jbeda** [comment]
I haven't used this process myself but this all looks sane to me.

**Comment by jbeda** [comment]
(I'm always confused if I'm supposed to use `lgtm` in this repo to signal technical review. I can't find that process written down anywhere -- root README.md, CONTRIBUTING.md or any of the obvious links off of those).

**Comment by heckj** [comment]
Thanks @jbeda 
/hold cancel
/approve

**Comment by opera443399** [comment]
in order to resolve conflicts now, redo the changes mentioned above. 
see #7370

**Comment by heckj** [comment]
closing this in favor of https://github.com/kubernetes/website/pull/7370, which appears to replace it.

**Pull request closed without merging**